.circleci/config.yml

```diff
@@ -78,13 +79,28 @@ jobs:
             npm i -g get-firefox
             get-firefox --branch nightly --platform linux --extract --target ~/
             ~/firefox/firefox -v
+            wget -O /tmp/geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v$GECKODRIVER/geckodriver-v$GECKODRIVER-linux64.tar.gz
```

Comment:
Is the version of geckodriver installed in `node_modules/.bin/geckodriver` usable for this? We're currently installed 0.21.0, but I don't know if that's intentional or if that was just the latest version when we set it up.